release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1749,7 +1844,17 @@ In the table below, features are marked with the following statuses:
 |CSI Azure File Driver Operator
 |-
 |TP
-|TP
+|GA
+
+|CSI Azure Stack Hub Driver Operator
```

Comment:
@lpettyjo, some of these were intentionally removed on https://github.com/openshift/openshift-docs/pull/48581. Please do not reintroduce them.

Reply:
@kalexand-rh I wasn't aware that we remove 3-GA items from the table (makes sense though). I've checked https://github.com/openshift/openshift-docs/pull/48581/ removals against my PR to ensure I wasn't adding back things that were removed.